_posts/2016-12-07-deprecating-nodes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Deprecating Nodes
+author: David Davis
+tags:
+  - release
+  - platform
+  - 3.0
+---
+
+Nodes were introduced in Pulp in 2.1.0 and provided a way to keep multiple Pulp
+servers synchronized. However, Nodes uses separate codepaths from the normal
+sync+publish codepaths which creates at a few problems:
+
+* Nodes is less tested than sync+publish even though it provides a similar
+feature set. This greatly reduces the user-to-user usage and testing benefit.
+* Fixing bugs in sync+publish does not fix bugs in Nodes and vice-versa
+* Nodes has a good amount of bugs already reported that are still unfixed
+
+Given that much of the functionality of Nodes can be achieved by running Pulp
+with sync+publish workflow, the core team is removing Nodes in the next major
+release of Pulp 3.0. With Pulp 2.12, Nodes will be officially deprecated.
+
+In this blog post, I want to outline how users can sync content across Pulp
+servers without using Nodes and how tools like cron and systemd can be used to
+keep content updated across Pulp servers. I am going to illustrate a two-server
+example in which I'll call one Pulp server the parent and the other the child.
+However, this distinction is arbitrary as both servers are each a standard Pulp
+server installation without any Nodes packages installed. The child Pulp server
+is called that because it is the one we want to mirror content to.
+
+Syncing Content to a Child Server
+---------------------------------
+
+Suppose I have an rpm repo called 'centos-base' on the parent Pulp server.
+First, I would want to publish that repository to make it available for the
+child Pulp server to sync:
+
+    pulp-admin rpm repo publish run --repo-id centos-base
+
+This repository should then be available via http or https. You can check its
+relative url via `pulp-admin rpm repo list --repo-id centos-base --details`. In
```

Comment:
As a small detail: you'll see a relative path via this command, not a URL. I think it's helpful for us to be careful and consistent about calling something a URL vs. a path, to avoid ambiguity.